src/Actions/index.ts

```diff
@@ -10,40 +10,13 @@ export const Update_Token_Action = function (token: string) {
   };
 };
 
-export const Update_Urls_List = function (urlsList: object[]) {
+export const UPDATE_URLS_LIST = function (urlsList: object[]) {
```

Comment:
array of Urls

Reply:
Did not got it 
tried but did not worked